CMakeLists.txt

```diff
@@ -50,217 +50,157 @@ add_library(nanostackeventloop STATIC)
 add_library(nanostackeventloopInterface INTERFACE)
 target_include_directories(nanostackeventloopInterface INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/nanostack-event-loop)
 
-include_directories(${NANOSTACK_LIBSERVICE_INCLUDE_DIR})
-
 target_sources(nanostackeventloop PRIVATE
     source/event.c
     source/event.h
     source/nanostack_eventloop.c
+    source/nanostack_eventloop.h
```

Comment:
Why are header-files added to nanostackeventloop sources?

Reply:
I was reverting changes from: https://github.com/PelionIoT/sal-stack-nanostack-eventloop/pull/51/files

But indeed those headers shouldnt be needed.